src/Aspire.Hosting.Kubernetes/KubernetesResource.cs

```diff
@@ -105,8 +110,9 @@ private void SetLabels()
     {
         Labels = new()
         {
-            ["app"] = "aspire",
-            ["component"] = resource.Name,
+            ["app.kubernetes.io/name"] = Parent.HelmChartName,
```

Comment:
Isn't this null by default? We should default to the app name. IHostingEnvironment.ApplicationName